### DISCUSSION TITLE
Loss value when training QueryInst

### DISCUSSION THREAD

**Discussion by bnbao** [open-mmlab/mmdetection#9072] [Q&A]
Hi all,
I really impressed with the work and the performance of QueryInst so I am tried to train it on my custom dataset. This dataset that I am successfully train on Swin Transformer. Unfortunately, I tried to train with Queryinstance with mmdetection so the loss values zero except loss_cls. Can you help me to solve that, I really appreciate it.

stage0_loss_cls: 0.0025, stage0_pos_acc0, stage4_loss_iou: 0.0000, stage4_loss_mask: 0.0000, stage5_loss_cls: 0.0024, stage5_pos_acc: 0.0000, stage5_loss_bbox: 0.0000, stage5_loss_iou: 0.0000, stage5_loss_mask: 0.0000,0.0000, stage1_loss_mask: 0.0000, stage2_loss_cls: 0.0012, stage2_pos_acc: 0.0000, stage2_loss_bbox: 0.0000, stage2_loss_iou: 0.0000, stage2_loss_mask: 0.0000, stage3_loss_cls: 0.0012, stage3_pos_acc: 0.0000, stage3_loss_bbox: 0.0000, stage3_loss_iou: 0.0000, stage3_loss_mask: 0.0000, stage4_loss_cls: 0.0010, stage4_pos_acc: 0.0000, stage4_loss_bbox: 0.000

looking for your reply!

**Comment by AwayFromKeyboardwarrior** [comment]
I am experiencing the same thing. did you happen to solve the problem? can you share please?
